packages/sanity/src/structure/components/paneItem/PaneItem.tsx

```diff
@@ -144,16 +139,22 @@ export function PaneItem(props: PaneItemProps) {
   // Reset `clicked` state when `selected` prop changes
   useEffect(() => setClicked(false), [selected])
 
+  // Preloads the edit state on hover, using concurrent rendering with `startTransition` so preloads can be interrupted and not block rendering
+  const [handleMouseEnter, preload] = usePreloadEditState(id, schemaType?.name, selected)
+
   return (
     <PreviewCard
       data-testid={`pane-item-${title}`}
       __unstable_focusRing
-      as={Link as FIXME}
+      as={ChildLink as FIXME}
+      // @ts-expect-error - `childId` is a valid prop on `ChildLink`
+      childId={id}
```

Comment:
I don't think we're guaranteed that `childId` is always a document id, so think we might end up preloading for preview items that's not documents.

I _think_ there should be a preview component that is only invoked in the context of a document, if so it would be better to move this logic there

Reply:
The hook requires a `documentType` to load, if `childId` isn't a document, will `schemaType?.name` still be a string?

Reply:
ok testing fix

Reply:
I think I fixed it, at least in my testing, can you confirm?

Reply:
Works now 🙌🏼